src/NuGet.Core/NuGet.Shared/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "version": "3.5.0-*",
-  "shared": "*.cs",
+  "shared": "**/*.cs",

```

Comment:
No, but I found a strange bug. Since there was a `Properties/AssemblyInfo.cs` which was not covered by the existing glob, `NuGet.Shared` produced a DLL. Since we were only using it as a `build` time dependency, this caused issues. It was unintentional that `NuGet.Shared` produce a DLL (I assume) so I changed the glob to ensure that it would never accidentally start producing a DLL.